src/Database/Concerns/HasRelationships.php

```diff
@@ -299,6 +321,7 @@ protected function handleRelation($relationName)
                 $relationObj = $this->$relationType($relation[0], $relation['public'], $relation['key'], $relationName);
                 break;
 
+            case 'hasOneThrough':
```

Comment:
Was this added to the database docs?

Reply:
@LukeTowers yep :) - https://github.com/octobercms/docs/blob/wip/laravel-6/database-relations.md#has-one-through

Reply:
Excellent, thanks!